packages/web-components/docs/components/pie.md

```diff
@@ -0,0 +1,56 @@
+# Data Commons Pie/Donut Chart Web Component
+
+[Data Commons Web Component](../../README.md) for visualizing multiple statistical variables around a single place on a pie/donut chart.
+
+## Usage
+
+```html
+<datacommons-pie
+  title="Median Income by gender in California"
+  place="geoId/06"
+  variables="Median_Income_Person_15OrMoreYears_Male_WithIncome Median_Income_Person_15OrMoreYears_Female_WithIncome"
+></datacommons-pie>
+```
+
+<img src="../assets/pie.png" width="620"/>
+
+### Attributes
+
+Required:
+
+- `date` _string_
+  - Specific date to show data for.
+  - Date should be formatted in ISO 8601 (e.g. "YYYY", "YYYY-MM", "YYYY-MM-DD")
+    and match the format used by the variable being plotted.
```

Comment:
Sorry, this section is a copy/paste mistake from my draft. Current pie charts don't have a date attribute. Let's remove until this gets implemented.

Reply:
Fixed!